src/init.js

```diff
@@ -122,30 +136,31 @@ on the option selected.\
 
       fs.makeTreeSync(packagePath);
 
-      const result = [];
       for (let childPath of Array.from(fs.listRecursive(templatePath))) {
         const templateChildPath = path.resolve(templatePath, childPath);
         let relativePath = templateChildPath.replace(templatePath, "");
         relativePath = relativePath.replace(/^\//, '');
+
+        // Files with `.template` extensions typically are named that way
+        // because they need part of their name replaced. Most other files that
+        // aren't named based on a parameter can omit the `.template`
+        // extension, even if their contents are partially parameterized.
         relativePath = relativePath.replace(/\.template$/, '');
         relativePath = this.replacePackageNamePlaceholders(relativePath, packageName);
 
         const sourcePath = path.join(packagePath, relativePath);
         if (fs.existsSync(sourcePath)) { continue; }
         if (fs.isDirectorySync(templateChildPath)) {
-          result.push(fs.makeTreeSync(sourcePath));
+          fs.makeTreeSync(sourcePath);
         } else if (fs.isFileSync(templateChildPath)) {
           fs.makeTreeSync(path.dirname(sourcePath));
           let contents = fs.readFileSync(templateChildPath).toString();
           contents = this.replacePackageNamePlaceholders(contents, packageName);
           contents = this.replacePackageAuthorPlaceholders(contents, packageAuthor);
           contents = this.replaceCurrentYearPlaceholders(contents);
-          result.push(fs.writeFileSync(sourcePath, contents));
-        } else {
-          result.push(undefined);
+          fs.writeFileSync(sourcePath, contents);
         }
       }
-      return result;
```

Comment:
So you did confirm that nobody uses this return value, right?

Reply:
Yeah, the only calls to `generateFromTemplate` come from within this file, and none of them do anything with the return value.